StreamChatClient_v3/Controllers/ChannelListController.swift

```diff
@@ -134,13 +139,47 @@ public class ChannelListControllerGeneric<ExtraData: ExtraDataTypes>: Controller
     }
 }
 
+// MARK: - Channel muting
+
+public extension ChannelListControllerGeneric {
+    typealias MuteChannelCompletion = (Error?) -> Void
```

Comment:
Please don't use typealiases for simple callbacks like this one. We use it only when the signature is more complex and it's used at multiple places. Using it like this makes reading the function signature more difficult because it's not clear what the actual signature is.

Reply:
@VojtaStavik done